exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -19,3 +19,20 @@
  * random (generated in function): 40
  * output: "Sum with value 70 isd left in 30 from number 100"
  */
+
+/**
+ * Generate a random number and add it to a given number, returning a message based on the sum.
+ * @param {number} value
+ * @returns {string}
+ */
+
+const getDistanceMessageFromSumTo100 = function(value) {
+    const randomNumber = Math.floor(Math.random() * 101); 
+    const sum = value + randomNumber;
+    if (sum > 100) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (sum > THRESHOLD) {
```

- [ ] Extract number to constant